src/config.ts

```diff
@@ -69,7 +69,7 @@ const getIssuerEndpoint = async (suffix?: string) => {
 
 const getPatientPickerEndpoint = async (suffix?: string) => {
     if (patientPickerEndpoint === defaultEndpoint) {
-        patientPickerEndpoint = await getIssuerEndpoint();
+        patientPickerEndpoint = `${await getIssuerEndpoint()}/v1`;
```

Comment:
This doesn't look like a valid URL to me. But I guess I need to see where it is being referenced.

Reply:
I think I can try to remove method getPatientPickerEndpoint